pennylane/math/multi_dispatch.py

```diff
@@ -870,6 +870,53 @@ def norm(tensor, like=None, **kwargs):
     return norm(tensor, **kwargs)
 
 
+@multi_dispatch(argnum=[0])
+def svd(tensor, like=None, **kwargs):
+    """Compute the singular value decomposition of a tensor in each interface.
+
+    For a matrix :math:`A`, the singular value decomposition consist of three matrices :math:`S`, :math:`U` and :math:`Vh`, such that:
+    .. math::

```

Comment:
```suggestion

    .. math::
```
Not currently showing up properly in the [docs](https://xanaduai-pennylane--5154.com.readthedocs.build/en/5154/code/api/pennylane.math.svd.html)